docs/sources/enterprise/reporting.md

```diff
@@ -120,6 +120,8 @@ When generating reports, each panel renders separately before being collected in
 
 To modify the panels' clarity you can set a scale factor for the rendered images. A higher scale factor is more legible but will increase the file size of the generated PDF.
 
+It's also possible to specify custom fonts supporting different Unicode scripts. DejaVu that is the default font used for PFD rendering.
```

Comment:
```suggestion
You can also specify a custom font that support different Unicode scripts. The DejaVu font is the default used for PDF rendering.
```

Reply:
Can a user specify "a" font or "any" font that fulfills that condition? I left it as "a" here because I assume that would otherwise require testing all all possible fonts.

Reply:
I wanted to stress that there's an option for specifying font different from the default one. The default font does not support, for example, Chinese glyphs. So having this option users are able to bypass the limitation of the default font by using some other one.